nohost/src/HTMLServer.js

```diff
@@ -34,6 +36,24 @@ define(function (require, exports, module) {
         this.fs = null;
     };
 
+    HTMLServer.prototype.readyToServe = function () {
+        var self = this;
+        var deferred = new $.Deferred();
+
+        // Decide if we can use Blob URLs or need to document.write()
+        Compatibility.supportsIFrameHTMLBlobURL(function(err, _shouldUseBlobURL) {
```

Comment:
nit: can we switch the naming convention for this? Have the global use `_` and this callback parameter should just be `shouldUseBlobURL`